webui/react/src/pages/ModelRegistry.tsx

```diff
@@ -422,17 +424,38 @@ const ModelRegistry: React.FC = () => {
   return (
     <Page docTitle="Model Registry" id="models">
       <Section title="Model Registry">
-        <ResponsiveTable
-          columns={columns}
-          dataSource={models}
-          loading={isLoading}
-          pagination={getFullPaginationConfig({
-            limit: settings.tableLimit,
-            offset: settings.tableOffset,
-          }, total)}
-          showSorterTooltip={false}
-          size="small"
-          onChange={handleTableChange} />
+        {models.length === 0 && !isLoading ?
```

Comment:
**suggestion:** place the `loading={isLoading}` attribute on the `Page` component, and so the children (empty state and responsive table) doesn't have to micro manage the `isLoading` bit.